package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "reactjsdenver.com",
```

Comment:
It can go anywhere and there's no clear standard. I have seen it right after the `name` key, which I think is a great place for it.